src/resource.jl

```diff
@@ -332,8 +332,30 @@ function serve_file(
     content_encoding::String;
     buffer::Vector{UInt8} = Vector{UInt8}(undef, 2*1024*1024),
 )
-    size = filesize(path)
-    HTTP.setheader(http, "Content-Length" => string(size))
+    content_length = filesize(path)
+    startbyte, stopbyte = 0, content_length-1
+
+    # Support single byte ranges
+    range_string = HTTP.header(http, "Range")
+    if !isempty(range_string)
+        m = match(r"^bytes=(\d*)-(\d*)$", range_string)
```

Comment:
```suggestion
        m = match(r"^\s*bytes\s*=\s*(\d*)\s*-\s*(\d*)\s*$", range_string)
```
Let's be a bit more liberal with allowing spaces. Are the zero-repetition options here intentional? I.e. is the logic that the regex allows no digits, which fails to parse below but it replaced by the default? If so, maybe a comment clarifying that. That took me a while to work out and I had to look at the examples to be sure that it was intentional.

Reply:
> Let's be a bit more liberal with allowing spaces.

Sure, I didn't see spaces in any examples but I guess it doesn't hurt.

> Are the zero-repetition options here intentional? I.e. is the logic that the regex allows no digits, which fails to parse below but it replaced by the default?

Yea, thought it was a cute way of doing it but maybe it is clearer to just check for a match.

Reply:
Eh, I guess one should `tryparse` anyway for safety? I will just add a comment.